packages/storybook/src/stories/Sidebar/Sidebar.stories.tsx

```diff
@@ -0,0 +1,171 @@
+import type { Meta } from '@storybook/react';
+import { withRouter } from 'storybook-addon-react-router-v6';
+
+import {
+	FileTextIcon,
+	FileCheckmarkIcon,
+	FolderMinusIcon,
+	TrashIcon,
+	CogIcon,
+	MagnifyingGlassIcon,
+	PlusIcon,
+	InboxFillIcon,
+} from '@navikt/aksel-icons';
+
+import { SidebarItem } from '../../../../frontend-design-poc/src/components/SidebarItem';
+import { Sidebar } from '../../../../frontend-design-poc/src/components/Sidebar';
+import { HorizontalLine } from '../../../../frontend-design-poc/src/components/Sidebar/Sidebar';
+
+const meta = {
+	title: 'Example/Sidebar',
+	component: Sidebar,
+	decorators: [withRouter],
+	tags: ['autodocs'],
+} satisfies Meta<typeof Sidebar>;
+
+export default meta;
+const SidebarSimpleExamplePerson = () => {
+	return (
+		<Sidebar>
+			<SidebarItem
+				displayText="Innboks"
+				label="Trykk her for å gå til innboks"
+				icon={<InboxFillIcon />}
+				count={3}
+				url="/innboks"
+				isInbox
+			/>
+			<HorizontalLine />
+			<SidebarItem
+				displayText="Nytt skjema"
+				label="Trykk her for å gå til utboks"
+				icon={<PlusIcon />}
+				url="/Nytt"
+				isButton
+			/>
+			<SidebarItem
+				displayText="Utkast"
+				label="Trykk her for å gå til Utkast"
+				icon={<FileTextIcon />}
+				count={8}
+				url="/Utkast"
+			/>
+			<SidebarItem
+				displayText="Sendt"
+				label="Trykk her for å gå til Sendt"
+				icon={<FileCheckmarkIcon />}
+				count={8}
+				url="/Sendt"
+			/>
+			<HorizontalLine />
+			<SidebarItem
+				displayText="Arkiv"
+				label="Trykk her for å gå til Arkiv"
+				icon={<FolderMinusIcon />}
+				count={8}
+				url="/Arkiv"
+			/>
+			<SidebarItem
+				displayText="Slettet"
+				label="Trykk her for å gå til Slettet"
+				icon={<TrashIcon />}
+				count={8}
+				url="/Slettet"
+			/>
+			<HorizontalLine />
```

Comment:
I don't think I am very fond this being a separate component, to be honest.
It sounds very generic, but still has a very important role in the design of the Sidde bar. I would it make more sense to be a part of the SidebarItem, enabled through props?

Reply:
Here I disagree with @seanes. I think its a good idea to keep it as its own component.

Reply:
Just to clearify: Making it prop makes sense because it's not applicable to the last element. 
But OK, not a biggie for me.